subprojects/docs/src/docs/userguide/dep-man/01-core-dependency-management/declaring_repositories.adoc

```diff
@@ -1,837 +1,346 @@
+The following example shows a build script that declares multiple patterns:
 
-Each `artifact` or `ivy` specified for a repository adds an _additional_ pattern to use. The patterns are used in the order that they are defined.
-
-.Ivy repository with multiple custom patterns
 ====
 include::sample[dir="snippets/artifacts/defineRepository/groovy",files="build.gradle[tags=ivy-repo-with-custom-pattern]"]
 include::sample[dir="snippets/artifacts/defineRepository/kotlin",files="build.gradle.kts[tags=ivy-repo-with-custom-pattern]"]
 ====
 
-Optionally, a repository with pattern layout can have its `'organisation'` part laid out in Maven style, with forward slashes replacing dots as separators. For example, the organisation `my.company` would then be represented as `my/company`.
+==== Specify a Maven Style Custom Pattern Layout
+
+Some Ivy repositories use Maven style for organisation layouts, where forward slashes replace dots as separators.
+For example, Maven style represents the organisation `my.company` as `my/company`.
+
+To use Maven style organization layouts in a repository, assign `m2compatible` a value of "true" in the `patternLayout` block:
 
-.Ivy repository with Maven compatible layout
 ====
 include::sample[dir="snippets/artifacts/defineRepository/groovy",files="build.gradle[tags=ivy-repo-with-m2compatible-layout]"]
 include::sample[dir="snippets/artifacts/defineRepository/kotlin",files="build.gradle.kts[tags=ivy-repo-with-m2compatible-layout]"]
 ====
 
 [[sub:accessing_secured_ivy_repositories]]
-=== Accessing authenticated Ivy repositories
+=== Access an Authenticated Ivy Repository
 
-You can specify credentials for Ivy repositories secured by basic authentication.
+You can specify credentials for Ivy repositories secured by basic authentication:
 
-.Ivy repository with authentication
 ====
 include::sample[dir="snippets/artifacts/defineRepository/groovy",files="build.gradle[tags=authenticated-ivy-repo]"]
 include::sample[dir="snippets/artifacts/defineRepository/kotlin",files="build.gradle.kts[tags=authenticated-ivy-repo]"]
 ====
 
-See <<#sec:supported_transport_protocols>> for authentication options.
+For additional authentication options, see <<repositories_cheat_sheet.adoc#sec:supported_transport_protocols,supported transport protocols>>.
 
 [[sec:repository-content-filtering]]
-== Repository content filtering
+== Filter Repository Contents
 
-Gradle exposes an API to declare what a repository may or may not contain.
-There are different use cases for it:
+Gradle exposes an API to control which dependencies your project can fetch from a repository.
+Common use cases include:
 
-- performance, when you know a dependency will never be found in a specific repository
-- security, by avoiding leaking what dependencies are used in a private project
-- reliability, when some repositories contain corrupted metadata or artifacts
+* performance (e.g. you know a dependency will never be found in a specific repository)
```

Comment:
question: I thought usage was `e.g.,` but divert to your knowledge there

Reply:
It's complicated, but I find it's helpful to think of it as "example given".